modules/Planner/resources_manage_editProcess.php

```diff
@@ -69,20 +69,20 @@
                 } else {
                     $row = $result->fetch();
 
-                    $type = $_POST['type'];
+                    $type = $_POST['type'] ?? '';
                     if ($type == 'File') {
                         $content = $row['content'];
                     } elseif ($type == 'HTML') {
-                        $content = $_POST['html'];
+                        $content = $_POST['html'] ?? '';
                     } elseif ($type == 'Link') {
-                        $content = $_POST['link'];
+                        $content = $_POST['link'] ?? '';
                     }
-                    $name = $_POST['name'];
-                    $category = $_POST['category'];
-                    $purpose = $_POST['purpose'];
+                    $name = $_POST['name'] ?? '';
+                    $category = $_POST['category'] ?? '';
+                    $purpose = $_POST['purpose'] ?? '';
                     $tags = strtolower($_POST['tags']);
-                    $gibbonYearGroupIDList = (!empty($_POST['gibbonYearGroupID']))? implode(',', $_POST['gibbonYearGroupID']) : '';
-                    $description = $_POST['description'];
+                    $gibbonYearGroupIDList = implode(',', $_POST['gibbonYearGroupID']) ?? '';
```

Comment:
This is outside the function call and will not have the desired effect. The null coalesce needs to be on the $_POST variable itself.